config/initializers/chartkick.rb

```diff
@@ -1,185 +1,186 @@
          }
       },
-      lineColor: '#707073',
-      minorGridLineColor: '#505053',
-      tickColor: '#707073',
+      lineColor: "#707073",
+      minorGridLineColor: "#505053",
+      tickColor: "#707073",
       title: {
          style: {
-            color: '#A0A0A3'
+            color: "#A0A0A3"
 
          }
       }
    },
    yAxis: {
-      gridLineColor: '#707073',
+      gridLineColor: "#707073",
       labels: {
          style: {
-            color: '#E0E0E3'
+            color: "#E0E0E3"
          }
       },
-      lineColor: '#707073',
-      minorGridLineColor: '#505053',
-      tickColor: '#707073',
+      lineColor: "#707073",
+      minorGridLineColor: "#505053",
+      tickColor: "#707073",
       tickWidth: 1,
       title: {
          style: {
-            color: '#A0A0A3'
+            color: "#A0A0A3"
          }
       }
    },
    tooltip: {
-      backgroundColor: 'rgba(0, 0, 0, 0.85)',
+      backgroundColor: "rgba(0, 0, 0, 0.85)",
       style: {
-         color: '#F0F0F0'
+         color: "#F0F0F0"
       }
    },
    plotOptions: {
       series: {
          dataLabels: {
-            color: '#0088CC'
+            color: "#000"
          },
          marker: {
-            lineColor: '#333'
+            lineColor: "#333"
          }
       },
       boxplot: {
-         fillColor: '#505053'
+         fillColor: "#505053"
       },
       candlestick: {
-         lineColor: 'white'
+         lineColor: "white"
       },
       errorbar: {
-         color: 'white'
+         color: "white"
       }
    },
    legend: {
       itemStyle: {
-         color: '#E0E0E3'
+         color: "#E0E0E3"
       },
       itemHoverStyle: {
-         color: '#FFF'
+         color: "#FFF"
       },
       itemHiddenStyle: {
-         color: '#606063'
+         color: "#606063"
       }
    },
    credits: {
       style: {
-         color: '#666'
+         color: "#666"
       }
    },
    labels: {
       style: {
-         color: '#0088CC'
+         color: "#0088CC"
       }
    },
 
    drilldown: {
       activeAxisLabelStyle: {
-         color: '#F0F0F3'
+         color: "#F0F0F3"
       },
       activeDataLabelStyle: {
-         color: '#F0F0F3'
+         color: "#F0F0F3"
       }
    },
 
    navigation: {
       buttonOptions: {
-         symbolStroke: '#DDDDDD',
+         symbolStroke: "#DDDDDD",
          theme: {
-            fill: '#505053'
+            fill: "#505053"
          }
       }
    },
    rangeSelector: {
       buttonTheme: {
-         fill: '#505053',
-         stroke: '#000000',
+         fill: "#505053",
+         stroke: "#000000",
          style: {
-            color: '#CCC'
+            color: "#CCC"
          },
          states: {
             hover: {
-               fill: '#707073',
-               stroke: '#000000',
+               fill: "#707073",
+               stroke: "#000000",
                style: {
-                  color: 'white'
+                  color: "white"
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.